packs/equipment-effects/effect-rhino-shot.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Rhino Shot",
     "system": {
         "description": {
-            "value": "<p>On a successful Strike, the attack deals an additional 2d6 force damage, and the target must succeed at a @Check[fortitude|dc:19] or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone]. If the hit with the <em>rhino shot</em> was a critical success, the target is knocked prone unless it critically succeeds at its save.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Rhino Shot]</p>\n<p>On a successful Strike, the attack deals an additional 2d6 force damage, and the target must succeed at a @Check[fortitude|dc:19] or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone]. If the hit with the <em>rhino shot</em> was a critical success, the target is knocked prone unless it critically succeeds at its save.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Rhino Shot]</p>\n<p>On a successful Strike, the attack deals an additional 2d6 force damage, and the target must succeed at a DC 19 Fortitude save or be knocked prone. If the hit with the <em>rhino shot</em> was a critical success, the target is knocked prone unless it critically succeeds at its save.</p>"
```

Reply:
After thinking about this, the automation should be moved to the Rhino Shot item itself - the effect isn't needed any longer. See Eroding Bullet.